src/pages/[id].page.jsx

```diff
@@ -29,20 +29,33 @@ export default function Details({ details }) {
   const [htmlCode, setHtmlCode] = useState('');
   const [cssCode, setCssCode] = useState('');
   const [javascriptCode, setJavascriptCode] = useState('');
+  const [canRenderSandpack, setCanRenderSandpack] = useState(false);
 
   // dynamic imports need to be asynchronous
   useEffect(() => {
-    const updateCodeBlocks = async () => {
-      const html = await import(`../code-examples/${slug}/html.js`);
-      setHtmlCode(html.default);
+    const updateSandpackExamples = async () => {
+      try {
+        const html = await import(`../code-examples/${slug}/html.js`);
+        setHtmlCode(html.default);
 
-      const css = await import(`../code-examples/${slug}/css.js`);
-      setCssCode(css.default);
+        const css = await import(`../code-examples/${slug}/css.js`);
+        setCssCode(css.default);
 
-      const javascript = await import(`../code-examples/${slug}/javascript.js`);
-      setJavascriptCode(javascript.default);
+        const javascript = await import(
+          `../code-examples/${slug}/javascript.js`
+        );
+        setJavascriptCode(javascript.default);
+        setCanRenderSandpack(true);
+        return;
+      } catch (e) {
+        // we want a console.log when there might not be any code
+        // examples, so I disabled this lint rule
+        // eslint-disable-next-line no-console
+        console.log(`We couldn't find code examples for ${slug}.`);
+        if (canRenderSandpack) setCanRenderSandpack(false);
```

Comment:
The only way `canRenderSandpack` would be true is if somehow `setCanRenderSandpack(true)` succeeded, but then something else threw an error. Since there's nothing else that would throw an error, we can delete this line entirely.

Reply:
Fixed, along with the comment above about the `return`: https://github.com/sparkbox/accessible-components/pull/132/commits/97780618f5d62f8d573941aa71667d91f85faa1c